datatypes/src/primitives/datetime.rs

```diff
@@ -430,19 +430,19 @@ impl DateTimeParseFormat {
         }
     }
 
-    fn has_tz(&self) -> bool {
+    pub fn has_tz(&self) -> bool {
```

Comment:
Gibt es hier einen anderen Weg als die Funktionen public zu machen? DateTimeParseFormat kommt in api::model vor

Reply:
The question is how i should convert the datatype crate type to the services crate type. For the conversion I need access to the fields fmt, has_tz, and has_time. In the datatype crate type the three fields and the corresponding getters are all private. So I could either make the getters public (what I did in the commit) to make them accessible in the services crate or I could make the fields public. What should I do in that case?

Reply:
in that case you can leave it like this